src/NServiceBus.Core/Serialization/SerializationFeature.cs

```diff
@@ -24,7 +24,9 @@ protected internal sealed override void Setup(FeatureConfigurationContext contex
             var mapper = new MessageMapper();
             var settings = context.Settings;
             var messageMetadataRegistry = settings.Get<MessageMetadataRegistry>();
-            mapper.Initialize(messageMetadataRegistry.GetAllMessages().Select(m => m.MessageType));
+            mapper.Initialize(messageMetadataRegistry.GetAllMessages()
```

Comment:
i dont think this has any impact since we also register types on demand in the MessageMetadataRegistry